app/controllers/top.coffee

```diff
@@ -12,10 +17,59 @@ Controller = Ember.Controller.extend
     "#{location.protocol}//www.gravatar.com/avatar/#{@get('user.gravatarId')}?s=48&d=mm"
   ).property('user.gravatarId')
 
+  broadcasts: (->
+
+    if @get('auth.signedIn')
+      broadcasts = Ember.ArrayProxy.create(
+        content: [],
+        lastBroadcastStatus: '',
+        isLoading: true
+      )
+      apiEndpoint = config.apiEndpoint
+      options = {}
+      options.type = 'GET'
+      options.headers = { Authorization: "token #{@auth.token()}" }
+
+      seenBroadcasts = Travis.storage.getItem('travis.seen_broadcasts')
+      seenBroadcasts = JSON.parse(seenBroadcasts) if seenBroadcasts
+
+      $.ajax("#{apiEndpoint}/v3/broadcasts", options).then (response) ->
+        if response.broadcasts.length
+          array = response.broadcasts.filter((broadcast) ->
+              unless broadcast.expired
+                if seenBroadcasts.indexOf(broadcast.id.toString()) == -1
+                  broadcast
+            ).map( (broadcast) ->
+              Ember.Object.create(broadcast)
+            ).reverse()
+          
+          if array.length
+            broadcasts.set('lastBroadcastStatus', array[0].category)
+        broadcasts.set('content', array)
+        broadcasts.set('isLoading', false)
+
+      broadcasts
+  ).property('broadcasts')
```

Comment:
This should probably not depend on itself, I think that it's better to only depend on `auth.signedIn` - if this changes, we should probably refresh the list.